### PR TITLE
separate storage on QField and QFC

### PR DESCRIPTION
**Pull request by lucienicolier**

### CODE REVIEW

File: documentation/how-to/projects.en.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ tx_slug: documentation_how-to_projects
 QField has a file selector that allows to open a project from the device locally. To open files from the cloud see [QFieldCloud ](../get-started/tutorials/get-started-qfc.md).
 
 !!! note
-    Starting with Android 11 apps are denied full access to main and external storage content. For QField, this means direct access to projects and datasets transferred and/or downloaded into storage folders is not possible anymore. You can know more [on this page](../get-started/storage.md).
+    Starting with Android 11 apps are denied full access to main and external storage content. For QField, this means direct access to projects and datasets transferred and/or downloaded into storage folders is not possible anymore. You can know more [on this page](../get-started/storage-qf.md).
```
Comment (suricactus) [inline documentation/how-to/projects.en.md:11]:
```suggestion
    Starting with Android 11 apps are denied full access to main and external storage content. For QField, this means direct access to projects and datasets transferred and/or downloaded into storage folders is not possible anymore. You can know more about [QFiled storage access](../get-started/storage-qf.md).
```

File: documentation/get-started/storage-qfc.en.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: Storage
+tx_slug: documentation_get-started_storage_qfieldcloud
+---
+
```
Comment (suricactus) [inline documentation/get-started/storage-qfc.en.md:5]:
```suggestion

QFieldCloud allows you to configure as much storage as you need for your project. 

```

**Review by suricactus — COMMENTED** [review]
A bit of intro for the qfc storage would be nice.

File: documentation/how-to/projects.en.md (hunk 1 above)
Comment (m-kuhn) [reply documentation/how-to/projects.en.md:11]: s/QFiled/QField

Comment (suricactus) [reply documentation/how-to/projects.en.md:11]: Thanks for spotting, fixed in the comment above.

**Comment by nirvn** [comment]
Will this link in QField need updating: https://github.com/opengisch/QField/blob/master/src/qml/QFieldLocalDataPickerScreen.qml#L567 ?

**Comment by lucienicolier** [comment]
> Will this link in QField need updating: https://github.com/opengisch/QField/blob/master/src/qml/QFieldLocalDataPickerScreen.qml#L567 ?

Good point! Yes, it will.

**Comment by nirvn** [comment]
I'm wondering if we should avoid changing the URL for QField storage when splitting the cloud part out. We've used that crucial link quite a lot when helping ppl deal with Android storage. Ppl on social media linked to it too.

**Comment by lucienicolier** [comment]
> I'm wondering if we should avoid changing the URL for QField storage when splitting the cloud part out. We've used that crucial link quite a lot when helping ppl deal with Android storage. Ppl on social media linked to it too.

I can keep the one for QField and only change the one for QFC

**Comment by nirvn** [comment]
@lucienicolier , that sounds like a better option to me, thanks.

**Review by suricactus — APPROVED** [review]

**Pull request merged**